gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -30,12 +30,12 @@ const NxToastPage = () =>
     <GalleryDescriptionTile>
       <NxP>
         Toasts are rendered on the top right corner of the screen, and will remain in place until the user
-        dismisses them individually. Each toast renders one <NxCode>NxAlert</NxCode>, which comes in four
-        variations: Error, Info, Warning, and Success. To find out more about <NxCode>NxAlert</NxCode>, you
-        can review the documentation <NxTextLink href="#/pages/Alert">here</NxTextLink>.
+        dismisses them individually. Each toast renders one
+        {' '}<NxTextLink href="#/pages/Alert"><NxCode>NxAlert</NxCode></NxTextLink>, which comes in four
```

Comment:
It works better when you nest code and text link the other way around.  I don't remember exactly what the problem was with doing it this way, but we found that there was one and documented on the `NxTextLink` page which is the preferred way to do it

Reply:
> It works better when you nest code and text link the other way around. I don't remember exactly what the problem was with doing it this way, but we found that there was one and documented on the `NxTextLink` page which is the preferred way to do it

updated here: [2659cb5](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/2659cb5b2b6545314d6e63ea8d416f7a38f3c22c)